src/collection/pages/main/loading.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react"
 import './loading.scss';
 
 const LoadingPage: React.FC = () => {
-    return <div className={'LoadingPageContainer'}></div>
+    return <div className={'LoadingPageContainer'}><div className={'LoadingPageBorder'}></div></div>
```

Comment:
Just curiosity, why we need `{}` instead of the plain attribute as below?
```suggestion
    return <div className="LoadingPageContainer"><div className="LoadingPageBorder"></div></div>
```

Reply:
It's just inertia 😅 `{}` is doesn't need in here